mne_bids/tests/test_utils.py

```diff
@@ -71,3 +76,89 @@ def test_age_on_date():
     assert age_on_date(bday, exp3) == 24
     with pytest.raises(ValueError):
         age_on_date(bday, exp4)
+
+
+def test_get_brainvision_paths():
+    """Test getting the file links from a BrainVision header."""
+    test_dir = _TempDir()
+    data_path = op.join(base_path, 'brainvision', 'tests', 'data')
+    raw_fname = op.join(data_path, 'test.vhdr')
+
+    with pytest.raises(ValueError):
+        _get_brainvision_paths(op.join(data_path, 'test.eeg'))
+
+    # Write some temporary test files
+    with open(op.join(test_dir, 'test1.vhdr'), 'w') as f:
+        f.write('DataFile=testing.eeg')
+
+    with open(op.join(test_dir, 'test2.vhdr'), 'w') as f:
+        f.write('MarkerFile=testing.vmrk')
+
+    with pytest.raises(ValueError):
+        _get_brainvision_paths(op.join(test_dir, 'test1.vhdr'))
+
+    with pytest.raises(ValueError):
+        _get_brainvision_paths(op.join(test_dir, 'test2.vhdr'))
+
+    # This should work
+    eeg_file_path, vmrk_file_path = _get_brainvision_paths(raw_fname)
+    head, tail = op.split(eeg_file_path)
+    assert tail == 'test.eeg'
+    head, tail = op.split(vmrk_file_path)
+    assert tail == 'test.vmrk'
+
+
+def test_copyfile_brainvision():
+    """Test the copying of BrainVision vhdr, vmrk and eeg files."""
+    output_path = _TempDir()
+    data_path = op.join(base_path, 'brainvision', 'tests', 'data')
+    raw_fname = op.join(data_path, 'test.vhdr')
+    new_name = op.join(output_path, 'tested_conversion.vhdr')
+
+    # IO error testing
+    with pytest.raises(ValueError):
+        copyfile_brainvision(raw_fname, new_name+'.eeg')
```

Comment:
whitespace around operator? not sure if this is a pep8 violation or just style

Reply:
mmmh I'd say it's a violation. Curiously, my linter did not catch it (linter-flake8 in atom)